tests/Traits/DataHelpers/ExportHelper.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Tests\Traits\DataHelpers;
+
+use FINDOLOGIC\FinSearch\Export\FindologicProductFactory;
+use PHPUnit\Framework\MockObject\MockObject;
+use Psr\Container\ContainerInterface as PsrContainerInterface;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepository;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\EntitySearchResult;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Shopware\Core\System\SalesChannel\SalesChannelEntity;
+use Shopware\Core\System\SystemConfig\SystemConfigCollection;
+use Shopware\Core\System\SystemConfig\SystemConfigEntity;
+
+trait ExportHelper
+{
+    public function getDefaultSalesChannelContextMock(): SalesChannelContext
+    {
+        $salesChannelId = Defaults::SALES_CHANNEL;
+
+        /** @var SalesChannelContext|MockObject $salesChannelContextMock */
+        $salesChannelContextMock = $this->getMockBuilder(SalesChannelContext::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        /** @var SalesChannelEntity|MockObject $salesChannelMock */
+        $salesChannelMock = $this->getMockBuilder(SalesChannelEntity::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $salesChannelMock->method('getId')->willReturn($salesChannelId);
+
+        $salesChannelContextMock->expects($this->any())
+            ->method('getContext')
+            ->willReturn($this->defaultContext);
+
+        $salesChannelContextMock->method('getSalesChannel')->willReturn($salesChannelMock);
+
+        return $salesChannelContextMock;
+    }
+
+    public function getContainerMock(array $repositories = []): PsrContainerInterface
```

Comment:
Add a typehint for the array values. This way we know what the array should actually contain.

```php
/**
 * @param SomeClass[] $repositories
 */
```